scripts/get_collection_ids.cdc

```diff
@@ -0,0 +1,15 @@
+import NonFungibleToken from "../../contracts/NonFungibleToken.cdc"
+import ExampleNFT from "../../contracts/ExampleNFT.cdc"
```

Comment:
I am thinking to make the scripts and transactions are generic and only based on the interfaces, If possible. Here we can provide the `CollectionPublicPath` as the parameter like a string and typecast to the publicPath is possible. 

Although I am not rigid about this just want to know the thoughts of the team. It is better to make it generalise so the other developers wouldn't need to change anything init.

Reply:
I can see pros and cons to either approach. Having generalized scripts in our repo that others can use across different implementations of `NonFungibleToken` would be great. The benefit of generalizability is paid for by passing the path. I think that's probably worth the benefit, especially if the path is a stored as a variable in the implementation contract as it's easily referenced.

Reply:
Yeah so do you mind updating the script to support the universal NFTs

Reply:
Made sure to add change to recent commits ✅

Reply:
Did you mean to remove this import? I noticed that the script no longer requires `ExampleNFT`.

Also, I think the generalized version is better as well!